homeassistant/components/axis/device.py

```diff
@@ -122,17 +110,17 @@ def option_video_source(self):
     @property
     def signal_reachable(self):
         """Device specific event to signal a change in connection status."""
-        return f"axis_reachable_{self.unique_id}"
+        return f"axis_reachable_{self._attr_unique_id}"
```

Comment:
Don't access the attributes for reading, use the public properties instead.
See note here: https://developers.home-assistant.io/docs/core/entity#entity-class-or-instance-attributes